examples/cluster/plot_linkage_comparison.py

```diff
@@ -33,28 +33,29 @@
 from sklearn import cluster, datasets
 from sklearn.preprocessing import StandardScaler
 
-np.random.seed(0)
-
 # %%
 # Generate datasets. We choose the size big enough to see the scalability
 # of the algorithms, but not too big to avoid too long running times
 
 n_samples = 1500
-noisy_circles = datasets.make_circles(n_samples=n_samples, factor=0.5, noise=0.05)
-noisy_moons = datasets.make_moons(n_samples=n_samples, noise=0.05)
-blobs = datasets.make_blobs(n_samples=n_samples, random_state=8)
-no_structure = np.random.rand(n_samples, 2), None
+seed = 170
```

Comment:
if it's a number, I'd just use the number in multiple places rather than having a variable (and I'd use 42 to feel better about it 😁  )

Reply:
I see, ok then, we can keep the 170.

Reply:
Great, variable removed in [5e4f187](https://github.com/scikit-learn/scikit-learn/pull/27153/commits/5e4f18789f8dcf8b1dfcc1700c7141470efec272), 170 kept as the number. It is a tricky balance with the seed values it seems :)